chipsec/modules/common/rtclock.py

```diff
@@ -34,7 +34,10 @@ def __init__(self):
         self.res = ModuleResult.PASSED
 
     def is_supported(self):
-        return self.cs.is_core()
+        if self.cs.is_core():
+            return True
+        self.res = ModuleResult.NOTAPPLICABLE
+        return False
```

Comment:
Need to do some more reading, however this looks like it's not implemented for atom

Reply:
This patch should not change the functionality, only the result. Do you prefer to find if is applicable to atom and make a single commit?

Reply:
Understand that functionality does not change.  The problem is the meaning of the results. 
 Not Applicable means that the test does not apply to a platform (registers don't exist or don't function in the same way).  

If you want to fix up the server piece that should be fine, but lets not categorize Atom until there's a definitive answer